solutions/standard/main.tf

```diff
@@ -463,11 +463,11 @@ module "code_engine_kibana" {
 
   apps = {
     (local.code_engine_app_name) = {
-      image_reference = "docker.elastic.co/kibana/kibana:${local.es_full_version}"
+      image_reference = var.kibana_image_digest != null ? "${var.kibana_registry_namespace_image}@${var.kibana_image_digest}" : (var.kibana_image_reference != null ? var.kibana_image_reference : "docker.elastic.co/kibana/kibana:${local.es_full_version}")
```

Comment:
This logic is not correct `kibana_image_reference` should no longer be used.

Reply:
Changed the logic.